app/frontend/src/registry/components/people/ApplicationAdd.vue

```diff
@@ -1,218 +1,156 @@
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </h5>
+      <p v-if="loading" class="card-text">
+        <b-row>
+          <b-col md="12">
+            <div class="fa-2x text-center">
+              <i class="fa fa-circle-o-notch fa-spin"></i>
             </div>
-            <div class="col-xs-12 col-sm-4 col-md-3 registry-item">
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.dewat"> Dewatering wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.rem"> Remediation wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.geo"> Geotechnical wells
-                </label>
-              </div>
-              <div class="checkbox">
-                <label>
-                  <input type="checkbox" v-model="editClassification.qualCodes.clos"> Closed-loop geoexchange wells
-                </label>
-              </div>
-            </div>
-          </div>
-        </form>
-      </div>
+          </b-col>
+        </b-row>
+      </p>
+      <p v-else class="card-text">
+        <b-form-group label="Certification">
+          <b-row>
+            <b-col md="2">Issued by</b-col>
+            <b-col md="3">
+                  <b-form-select id="issuer" :options="formOptions.issuer" v-model="qualificationForm.primary_certificate" required></b-form-select>
+            </b-col>
+            <b-col md="2">Certificate number</b-col>
+            <b-col md="3">
+              <b-form-input type="text" placeholder="Enter certificate number" v-model="qualificationForm.primary_certificate_no" required></b-form-input>
+            </b-col>
+          </b-row>
+        </b-form-group>
+        <b-row>
+          <b-col md="2">Select classification</b-col>
+          <b-col md="8">
+            <b-form-radio-group class="fixed-width" :options="formOptions.classifications" @change="changedClassification" v-model="qualificationForm.subactivity" required></b-form-radio-group>
+          </b-col>
+        </b-row>
+        <b-form-group label="Qualified to drill">
+          <b-row>
+            <b-col md="8">
+              <b-form-checkbox-group class="fixed-width" :options="formOptions.qualifications" v-model="qualificationForm.qualifications" disabled>
+              </b-form-checkbox-group>
+            </b-col>
+          </b-row>
+        </b-form-group>
+        <b-form-group label="Date application received">
+          <b-row>
+            <b-col>
+              <datepicker format="yyyy-MM-dd" v-model="qualificationForm.status_set[0].effective_date" required></datepicker>
```

Comment:
I don't know why I'm not just using <b-form-input v-model="qualificationForm.status_set[0].effective_date" type="date" required/>, I'll make that change.